MCprep_addon/world_tools.py

```diff
@@ -151,72 +160,72 @@ def convert_mtl(filepath):
 	alpha maps and what not, which causes issues in ACES and whatnot where
 	non-color data is not an option.
 
+	This MTL conversion simply does the following:
+	- Comment out lines that begin with map_d
+	- Add a header at the end
+
 	Returns:
-		True if success, False if failed
+		True if success or skipped, False if failed, or None if skipped
 	"""
-	blender_standard = (
-		"Standard",
-		"Filmic",
-		"Filmic Log",
-		"Raw",
-		"False Color"
-	)
 	mtl = filepath.rsplit(".", 1)[0] + '.mtl'
 	lines = None
 	copied_file = None
 	with open(mtl, 'r') as mtl_file:
 		lines = mtl_file.readlines()
 
-	if bpy.context.scene.view_settings.view_transform not in blender_standard:
-		# This represents a new folder that'll backup the MTL filepath
-		original_mtl_path = Path(filepath).parent.absolute() / "ORIGINAL_MTLS"
-		# TODO: make sure this works in 2.7x. It should since 2.8 uses 3.7 but
-		# we should confirm nonetheless
-		original_mtl_path.mkdir(parents=True, exist_ok=True)
+	if bpy.context.scene.view_settings.view_transform in BUILTIN_SPACES:
+		return None
 
-		mcprep_header = (
-			"# This section was created by MCprep's MTL conversion script\n",
-			"# Please do not remove\n",
-			"# Thanks c:\n"
-		)
+	# This represents a new folder that'll backup the MTL filepath
+	original_mtl_path = Path(filepath).parent.absolute() / "ORIGINAL_MTLS"
```

Comment:
@StandingPadAnimations is there a reason to use this over something more like:

```suggestion
	original_mtl_path = Path(os.path.join(filepath, "ORIGINAL_MTLS")).parent.absolute()
```

I just worry about the slash being OS dependent. But if it worked for you (on a windows machine?) then I'm good with it.

Reply:
And I did test on 2.7x, works fine.

Reply:
Well I tested my change above, and it actually breaks the unit tests (a sign they are working), not sure exactly why though, I'm also just less familiar with the Path module.

So, really the main thing is if you could confirm you were testing on windows, then I'm all set.

Reply:
I'm on Linux but the [Python documentation](https://docs.python.org/3/library/pathlib.html#operators) says that it should be fine.

Reply:
Thanks for the doc reference, good to know. Unless you have any particular complaints here, I'll go ahead and merge in a moment.